src/ControlSystem/SettleToConstant.hpp

```diff
@@ -0,0 +1,62 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <vector>
+
+#include "ControlSystem/FunctionOfTime.hpp"
+#include "DataStructures/DataVector.hpp"
+
+/// \ingroup ControlSystemGroup
+/// Contains functions of time to support the dual frame system.
+namespace FunctionsOfTime {
+/// \ingroup ControlSystemGroup
+/// Sets f(t) and derivatives via exponentials, such that f(t)
+/// approaches a constant in time.
+/// Function is f(t) = A + (B+C(t-t0)) exp(-(t-t0)/tau)
+/// where A,B,C,t0,tau are constants.
```

Comment:
/// where A is the target value, t0 is the match time, tau is the decay time, and B and C are constants (determined by the matching).